AndelaOSP/Andela-Socials
# #161058409 Setup frontend test environment

#### What Does This PR Do?
> Add configuration for circle ci to run frontend test 
#### Description Of Task To Be Completed
- [x] update circleci yml file to run frontend test
- [x] Add lint and husky script to package.json
- [x] update the lint-staged script in the package.json
#### Any Background Context You Want To Provide?
> The coverage report is a combination of both the frontend and backend test but the checking of vulnerabilities has only the been done for the frontend.
#### How can this be manually tested?
>*  Fetch the branch locally by running `git fetch origin ch-setup-unit-test-161058409` 
>*  Checkout to branch `git checkout  ch-setup-unit-test-161058409`
>*  To lint check the frontend run `yarn --cwd client/package.json lint` from the root or cd into the client folder and run `yarn lint`
>* To fix lint errors run `yarn --cwd client/package.json lint --fix` from the root or cd into the client folder and run `yarn lint --fix`
#### What are the relevant pivotal tracker stories?
[#161058409](https://www.pivotaltracker.com/story/show/161058409)
#### Screenshot(s)
N/A
